custom-namespaces.yaml

```diff
@@ -1348,23 +1348,23 @@ websites:
     moz_org_page_metrics:
       type: table_view
       tables:
-        - table: moz-fx-data-marketing-prod.ga_derived.www_site_page_metrics_v1
+        - table: moz-fx-data-marketing-prod.ga.www_site_page_metrics
```

Comment:
Should those views be visible and accessible via mozdata project instead?

Reply:
I think because this is in the marketing project, the views are in the marketing project also, not in mozdata

Reply:
Let me rephrase the question, apart for the reason of how the ingestion of this data is set up. Is there a specific reason why this data is accessible via the marketing project vs out standardised way via moz-prod and mozdata projects?

Reply:
I think mainly because it is marketing data, and that is where the prior data lived also, so for sake of keeping things findable, we kept the GA4 data in the same spot as the GA3 data